api/alphacc_config.cc

```diff
@@ -70,12 +73,13 @@ bool ParseAlphaCCConfig(const std::string& file_path) {
       GetString(second, "onnx_model_path", &config->onnx_model_path));
   second.clear();
 
-  RETURN_ON_FAIL(GetValue(top, "redis", &second));
-  RETURN_ON_FAIL(GetString(second, "ip", &config->redis_ip));
-  RETURN_ON_FAIL(GetInt(second, "port", &config->redis_port));
-  RETURN_ON_FAIL(GetString(second, "session_id", &config->redis_sid));
-  RETURN_ON_FAIL(GetInt(second, "redis_update_duration",
-                        &config->redis_update_duration_ms));
+  if (GetValue(top, "redis", &second)) {

```

Comment:
I think we should directly remove all code related Redis. I confirmed with our team member that these code will never be  reused forever.

Reply:
Done